### PR TITLE
Add check for no argumetn in callback in describes

### LINKED ISSUE #14: Detecting describe(function (done) { ... })
I originally reported an issue in Jasmine: https://github.com/jasmine/jasmine/issues/896 but now I think it would be actually more appropiate here.

> I spent half an hour debugging this.

>     describe('stuff', function (done) {
>         it('works good', function () {
>             myHttp.get(url).then(function (response) {
>                 expect(response.data.length).toBeGreaterThan(10000);
>             }).then(done, fail);
>         });
>     });

> I got an error in some other unrelated test, with no stacktrace (because async), and had to fire up a debugger to understand what's happening. And it was just a simple mistake.

> (In case you still didn't notice: I put `done` in `describe` instead of `it`.)

> I believe this wouldn't be hard to do (as you already look at the arguments of the function given in `it`) and would save other programmers from mistakes like this.

### PR DESCRIPTION
**Pull request by SimenB**
Fixes #14

Seems like a low-hanging fruit :smile: 

I've never written a rule for eslint before, so this might be wrongly implemented

### CODE REVIEW

**Comment by tlvince** [comment]
This is looking good. Thanks Simen.

A few points:

* It was a little unclear to me solely by the name (`describe-with-done`) what the intentions of the rule were. Do you think `no-describe-args` is clearer? (I thought the comment `Enforce that a describe's callback does not contain any arguments` was spot on)
* Could you format the Git commit message as per the [AngularJS conventions](https://github.com/ajoslin/conventional-changelog/blob/master/conventions/angular.md)? This is so a new release will be published automatically by [semantic-release](https://github.com/semantic-release/semantic-release). This would be a `feat` type.
* Also, per [contributing](https://github.com/tlvince/eslint-plugin-jasmine/blob/master/CONTRIBUTING.md), could you write a brief doc?

If you could address those and squash the commit, that'd be awesome. Thanks for you contributions.

**Comment by SimenB** [comment]
I went with `no-suite-callback-args`, is that OK?

I completely forgot about the docs, sorry about that :laughing: 

Should be GTG now.

**Comment by SimenB** [comment]
@tlvince A thought, should I check in `fdescribe`s and `xdescribe`s as well? `fdescribe` is an error as well, and while `xdescribe` would error, add it for consistency

File: test/rules/no-suite-callback-args.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var rule = require('../../lib/rules/no-suite-callback-args');
+var RuleTester = require('eslint').RuleTester;
+
+var eslintTester = new RuleTester();
+
+eslintTester.run('describe-with-done', rule, {
```
Comment (SimenB) [inline test/rules/no-suite-callback-args.js]: might be overkill for the tests, I just added every sample I put in the docs

**Comment by tlvince** [comment]
Excellent! :+1:

**Comment by tlvince** [comment]
Landed: https://github.com/tlvince/eslint-plugin-jasmine/releases/tag/v1.5.0

**Comment by mik01aj** [comment]
Great! thanks! :+1:

**Pull request merged**